components/specification/transforms/2016-DEV0-to-2015-01.xsl

```diff
@@ -33,10 +33,6 @@
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
     xmlns:OME="http://www.openmicroscopy.org/Schemas/OME/2016-DEV0"
-    xmlns:Bin="http://www.openmicroscopy.org/Schemas/BinaryFile/2016-DEV0"
-    xmlns:SPW="http://www.openmicroscopy.org/Schemas/SPW/2016-DEV0"
-    xmlns:SA="http://www.openmicroscopy.org/Schemas/SA/2016-DEV0"
-    xmlns:ROI="http://www.openmicroscopy.org/Schemas/ROI/2016-DEV0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Removing the elements from the exclusion list is the real fix, it seems.  I've pushed a new commit which makes the namespace prefixes get unified to use `OME:` on upgrade and restored to their separate prefixes on downgrade.